OpenChange/MAPIStoreMailMessage.m

```diff
@@ -1394,6 +1394,28 @@ - (int) getPidLidMeetingType: (void **) data
           : MAPISTORE_ERR_NOT_FOUND);
 }
 
+- (int) getPidTagTransportMessageHeaders: (void **) data
+                    inMemCtx: (TALLOC_CTX *) memCtx
+{
+  NSDictionary *allHeaders = [sogoObject mailHeaders];
```

Comment:
I'd rather like having declarations as clean as possible to improve readability and I think it is consistent across the `sogo-openchange` code. So I will keep the variable declaration ahead and then the first assignment as follows:
```Objective-C
NSDictionary *allHeaders;
NSString *format, *headersString;
NSUInteger i, lastHeader;

headersString = @"";
format = @"%@: %@\n"
lastHeader = [allHeaders count] - 1
```